src/main/java/seedu/address/logic/command/sort/SortAccommodationCommand.java

```diff
@@ -0,0 +1,84 @@
+package seedu.address.logic.command.sort;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.Comparator;
+
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
+import seedu.address.logic.command.CommandResult;
+import seedu.address.logic.command.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.accommodation.Accommodation;
+import seedu.address.model.accommodation.UniqueAccommodationList;
+import seedu.address.model.travelplan.AccommodationList;
+import seedu.address.model.travelplan.TravelPlan;
+
+
+public class SortAccommodationCommand extends SortCommand {
+    public static final String COMMAND_WORD = "sort accommodation";
```

Comment:
Is this used anywhere? Same for all the other tpo ones.

Reply:
ya its used in the success message i think